extensions/spark/kyuubi-spark-connector-tpcds/src/test/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalogSuite.scala

```diff
@@ -48,6 +50,20 @@ class TPCDSCatalogSuite extends KyuubiFunSuite {
     assert(spark.sql(s"SHOW NAMESPACES IN tpcds.sf1").collect().length == 0)
   }
 
+  test("exclude databases") {
+    Seq("TINY,sf10" -> 2, "sf1 , " -> 1, "none" -> 0).foreach {
+      case (excludeDatabases, excludeNum) =>
+        val sparkConf = new SparkConf().setMaster("local[*]")
+          .set("spark.ui.enabled", "false")
+          .set("spark.sql.catalogImplementation", "in-memory")
+          .set("spark.sql.catalog.tpcds", classOf[TPCDSCatalog].getName)
+          .set("spark.sql.catalog.tpcds.excludeDatabases", excludeDatabases)
+        withSparkSession(SparkSession.builder.config(sparkConf).getOrCreate()) { spark =>
+          assert(spark.sql(s"SHOW DATABASES IN tpcds").count == 11 - excludeNum)
+        }
```

Comment:
It is more accurate to check whether the result database does not contain excludeDatabases

Reply:
updated